goal_src/jak1/pc/progress-pc.gc

```diff
@@ -2906,7 +2906,7 @@
 ;; override the enter handler for progress-going-out
 (set! (-> progress-going-out enter)
       (lambda :behavior progress ()
-        (sound-play "menu-close")
+        (sound-play "menu-close" :vol (-> *setting-control* default sfx-volume)) ;; og:preserve-this Make this sound respect sfx setting
```

Comment:
Don't need a `preserve-this` comment because the `pc` files are not touched by the decompiler, but not a big deal if it stays.

Reply:
Can clean it up while I'm here but yeah, skip preserves in files that aren't generated by the decompiler makes sense whoops.